.gitignore

```diff
@@ -3,9 +3,6 @@ __pycache__/
 *.py[cod]
 *$py.class
 
-# Personal configuration
-.vscode
```

Comment:
I don't think we want to track this.  What if I want a custom keyboard shortcut that you don't like?